bundle/manifests/broker.amq.io_activemqartemises.yaml

```diff
@@ -1970,8 +1974,6 @@ spec:
                         type: integer
                       grpc:
                         description: GRPC specifies an action involving a GRPC port.
```

Comment:
What is this related to?

Reply:
This new field is auto generated because of our dependency upgrade (which is needed by the cert manager package dependency)